modules/Bio/EnsEMBL/Variation/Pipeline/AncestralAlleles/AncestralAlleles_conf.pm

```diff
@@ -61,17 +61,16 @@ sub default_options {
         hive_use_triggers => 0,
         hive_auto_rebalance_semaphores => 0, 
         hive_no_init => 0,
+
         # the location of your checkout of the ensembl API (the hive looks for SQL files here)
-        
-        ensembl_cvs_root_dir    => $ENV{'HOME'} . '/bin',
-        hive_root_dir           => $ENV{'HOME'} . '/bin/ensembl-hive', 
+        ensembl_cvs_root_dir    => $ENV{'ENSEMBL_CVS_ROOT_DIR'} || '/hps/software/users/ensembl/repositories/' . $ENV{'USER'},
```

Comment:
Could $self->o('ensembl_cvs_root_dir') be used instead of file path.
Suggest maybe using $ENV{'ENSEMBL_ROOT_DIR'} in future as used in "Codon QuickStart: Running Pipelines"